EXILED/Exiled.API/Features/Player.cs

```diff
@@ -687,24 +687,24 @@ public ScpSpawnPreferences.SpawnPreferences ScpPreferences
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
         // TODO: Change logic for FacilityGuard in next major update
-        public bool IsNTF => Role?.Team is Team.FoundationForces;
+        public bool IsNTF => Role?.Type.IsNtf() ?? false;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleTypeId"/> is any Chaos rank.
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
-        public bool IsCHI => Role?.Team is Team.ChaosInsurgency;
+        public bool IsCHI => Role?.Type.IsChaos() ?? false;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleTypeId"/> is any SCP.
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
-        public bool IsScp => Role?.Team is Team.SCPs;
```

Comment:
Revert this change it's bad in my opinion

Reply:
How so?

Reply:
Also I was told to do this again, didn't do it as part of my own will. @moddedmcplayer what do you say about this review?

Reply:
> Revert this change it's bad in my opinion

What we had before was duplicate code, even if it looks slightly better.

Reply:
> > Revert this change it's bad in my opinion
> 
> What we had before was duplicate code, even if it looks slightly better.

it's still better to keep this one at what it's was before

Reply:
So what is it that we are doing then?

Reply:
@louis1706 @moddedmcplayer  please respond to the above ^^
I don't see the point in removing this code as now it fits better, we are now using exiled code to get these values
so I dont think we should revert this change lol

Reply:
> it's still better to keep this one at what it's was before

why exactly?